chrome/locale/en-US/zotero/preferences.ftl

```diff
@@ -36,3 +36,10 @@ preferences-color-scheme-dark =
 preferences-advanced-language-and-region-title = Language and Region
 preferences-advanced-enable-bidi-ui =
     .label = Enable bidirectional text editing utilities
+
+preferences-plus-btn =
+    .aria-label = Add
+    .label = +
+preferences-minus-btn =
+    .aria-label = Remove
+    .label = -
```

Comment:
- Include zotero.ftl and use `{ general-add }`/`{ general-remove }`.
- We don't want `+` and `-` to appear as localizable strings. If you omit `.label` here, does it overwrite the hard-coded `label`? (I'm not sure why it would.)

Reply:
> Include zotero.ftl and use { general-add }/{ general-remove }

Got it - repushed.

> We don't want + and - to appear as localizable strings. If you omit .label here, does it overwrite the hard-coded label? (I'm not sure why it would.)

It does do that... Despite my expectation, adding explicit `data-l10n-attrs="aria-label"` does not help either. It's something I ran into with #3987 as well where as a clunky workaround I had to save the label, add `data-l10n-id` and re-add the label after. We try something similar here but it's not unnecessarily complex - it'll look at different fluent use cases in firefox to see if there is a solution there

Reply:
So far, the least hacky way I discovered for here and the advanced search window (these are the only 2 instances where we need to add localization to something that already have a label) is to manually fetch the localized strings:
```
document.l10n.formatValues(['general-add', 'general-remove'])
	.then((res) => {
		let [add, remove] = res;
		document.querySelector("#quickCopy-delete").setAttribute('aria-label', remove);
		document.querySelector("#quickCopy-add").setAttribute('aria-label', add);
	});
```
While not pretty, it at least does not feel as hacky as my initial idea. Does it make sense? I'm not sure how many of such cases we are going to have. If it's just here and #3987, this is probably not an issue. But if this is something that we would encounter more often, we can move an abstraction of this into one of the JS files that's loaded for every window (customElements.js?) and expose this functionality via our own `data-l10n-` attributes.